CMakeLists.txt

```diff
@@ -40,18 +40,24 @@ option(SWIFT_BUILD_TOOLS
     "Build the Swift compiler and other tools"
     TRUE)
 
-option(SWIFT_BUILD_STDLIB
-    "Build the Swift standard library (independent of the SDK headers)"
-    TRUE)
-
 option(SWIFT_BUILD_SDK_OVERLAY
     "Build Swift SDK overlay"
     TRUE)
 
+option(SWIFT_BUILD_DYNAMIC_STDLIB
+    "Build dynamic variants of the Swift standard library and SDK overlay"
+    TRUE)
+
 option(SWIFT_BUILD_STATIC_STDLIB
     "Build static variants of the Swift standard library and SDK overlay"
     FALSE)
 
+if(SWIFT_BUILD_DYNAMIC_STDLIB OR SWIFT_BUILD_STATIC_STDLIB)
+  set(SWIFT_BUILD_STDLIB TRUE)
+else()
+  set(SWIFT_BUILD_STDLIB FALSE)
+endif()
```

Comment:
So setting `SWIFT_BUILD_STDLIB` like this would not be correct, since in the build that just builds the SDK overlay both `SWIFT_BUILD_DYNAMIC_STDLIB` and `SWIFT_BUILD_STATIC_STDLIB` would be set.